eng/testing/performance/android_scenarios.proj

```diff
@@ -1,4 +1,9 @@
 <Project Sdk="Microsoft.DotNet.Helix.Sdk" DefaultTargets="Test">
+  <PropertyGroup>
+      <IncludeXHarnessCli>true</IncludeXHarnessCli>
+      <MicrosoftDotNetXHarnessCLIVersion>1.0.0-prerelease.21566.2</MicrosoftDotNetXHarnessCLIVersion>
```

Comment:
You shouldn't need to set `MicrosoftDotNetXHarnessCLIVersion` as it should come from https://github.com/dotnet/runtime/blob/main/eng/Versions.props